ci/helm-chart/values.yaml

```diff
@@ -33,6 +33,8 @@ podAnnotations: {}
 podSecurityContext: {}
   # fsGroup: 2000
 
+priorityClassName: ""
```

Comment:
The if statement which evaluates if this is populated or not in deployment.yaml will evaluate to true and populate this invalid null string by default won't it?

Should this be commented out by default?

Reply:
Never mind docs say empty strings evaluate false.

https://helm.sh/docs/chart_template_guide/control_structures/

Reply:
TIL! So do you think this is good to merge?